docs/syntax-reference.rst

```diff
@@ -247,17 +247,21 @@ The WITH clause supports the following properties:
 |              | set, then the format of the input stream/table is     |
 |              | used.                                                 |
 +--------------+-------------------------------------------------------+
-| PARTITIONS   | The number of partitions in the topic. If this        |
-|              | property is not set, then the number of partitions of |
-|              | the input stream/table will be used.                  |
+| PARTITIONS   | The number of partitions in the topic.                |
+|              | If this property is not set, then the number of       |
+|              | partitions is taken from the value of the             |
+|              | ``ksql.sink.partitions`` property.                    |
+|              | ``ksql.sink.partitions`` can be set in the properties |
+|              | file the KSQL server is started with, or by using the |
+|              | ``SET`` statement.                                    |
```

Comment:
You may want to also mention that the default value for `ksql.sink.partitions` is 4.

Reply:
Good call. Consider it done.